resources/building-mobile-extensions/event-processing.md

```diff
@@ -1,42 +1,44 @@
 # Event processing
 
-An event is composed of the following information:
+An event is composed of the following components:
```

Comment:
Maybe "fields" or "parameters" is a better word here? Check with @emdobrin.

Reply:
I think `components` is ok here, as we are not defining the field names (as defined in code) in this document and it is a high level overview